ecl/eclagent/eclagent.hpp

```diff
@@ -25,6 +25,8 @@ version: "1.0"
     name: myeclagent
     thorConnectTimeout: 600
     traceLevel: 0
+    costs:
```

Comment:
@richardkchapman should we be adding commented out and documented items into values.yaml for each value that makes sense to be overridden?

Reply:
This file was deleted in HPCC-23609 and the yaml definition moved to ecl/agentexec/agentexec.cpp.